src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    private String color;
```

Comment:
You already have color in the parent class Figure, let's remove this field from the child classes